asv_bench/benchmarks/tslibs/tslib.py

```diff
@@ -21,7 +21,10 @@
 import numpy as np
 import pytz
 
-from pandas._libs.tslib import ints_to_pydatetime
+try:
```

Comment:
i think generally we should add a comment on what version these are changed so when we know when to remove the try/except

Reply:
isnt the idea to be able to run these benchmarks on older pandas versions basically indefinitely?

Reply:
sure, the reason I think we should add the version number is that so we can eventually remove these try/excepts, otherwise they will stay forever which is pretty cluttering. anyhow for another issue.